.github/workflows/compliance.yml

```diff
@@ -1,7 +1,7 @@
 name: "PR Compliance"
 
 on:
-  pull_request_target:
```

Comment:
I don't know why you use `pull_request_target`, but it does **_not_** actually run test for the contents of PR...

Reply:
Thanks. This was an oversight.